templates/artist/includes/artist_detail_updates.html

```diff
@@ -1,56 +1,59 @@
 <div class="tabs-panel {% if form.errors %}is-active{% endif %}" id="updates">
-    {% if updates %}
-        <h2>UPDATES</h2>
-        <div class="break">
-            <hr>
-        </div>
-        {% include "artist/includes/artist_updates.html" with updates=updates has_permission_to_submit_update=has_permission_to_submit_update %}
-    {% endif %}
+    {% if user.is_authenticated %}
+        {% if updates or has_permission_to_submit_update %}
+            <h3>LATEST HAPPENINGS</h3>
+            {% include "artist/includes/artist_updates.html" with updates=updates has_permission_to_submit_update=has_permission_to_submit_update %}
 
-    {% if has_permission_to_submit_update %}
-        <h2>POST AN UPDATE</h2>
-        <form action="" method="POST" enctype="multipart/form-data">
-            {% csrf_token %}
-            <p>{{ form.non_field_errors }}</p>
-            <p>
-                {{ form.title.errors }}
-                {{ form.title.label }}:
-                {{ form.title }}
-            </p>
-            <p>
-                {{ form.text.errors }}
-                {{ form.text.label }}:
-                {{ form.text }}
-            </p>
+            <h2>POST AN UPDATE</h2>
```

Comment:
Nope, this isn't right. Now you just added the form to submit an update for _every_ user, so long as at least one update exists. The update form needs to have `{% if has_permission_to_submit_update %}` surrounded around it, like in the way the code currently looks on master.